R-package/R/package.R

```diff
@@ -0,0 +1,29 @@
+
+#-------------------------
+# Load the python-modules
+#-------------------------
+
+
+RGF_mod <- NULL; RGF_utils <- NULL; SCP <- NULL;
+
+
+.onLoad <- function(libname, pkgname) {
+
+  if (reticulate::py_available(initialize = TRUE)) {
+
+    if (reticulate::py_module_available("rgf.sklearn")) {
```

Comment:
Will this function raise understandable error about the need of `rgf` installation? If not, then I think you should write `else` branch and do it there manually.

Reply:
If you don't mind I would rather keep this file intact. Otherwise it might lead to problems when I have to check-test the package on [CRAN](https://cran.r-project.org/)

Reply:
Got it!